options.go

```diff
@@ -77,6 +84,19 @@ func checkOptions(rp *Ringpop) []error {
 
 // Runtime options
 
+// Clock is used to set the Clock mechanism.  Testing harnesses will typically
+// replace this with a mocked clock.
+func Clock(c clock.Clock) Option {
+	return func(r *Ringpop) error {
+		if c == nil {
+			return errors.New("Clock is required")
```

Comment:
By convention, error messages in Golang do not start with a capital letter (unless you're referring to a name I guess). Just an FYI, not a blocker, if that's what you intended.